### ISSUE TITLE
Step 3 / Moving from Clover to OpenCore???

### ISSUE THREAD

**Issue by DanielYawJr** [tylernguyen/x1c6-hackintosh#74]
This is my first time making a Hackintosh - I'm excited to be using MacOS on my Lenovo X1 Carbon 6th gen! I'm trying to use MacOS as my exclusive operating system, and completely remove Win10. 

I'm a bit confused about step 3, where we move away from booting off the USB. 

I'm not sure about what it means to install OpenCore on the EFI? 

Here's what I've done so far:

1. Created MacOS Install USB
2. Swapped Wifi cards to a Fenvi one
3. Installed MacOS
4. Now I boot into MacOS using the Clover USB
5. Used OpenEFI to mount hard drive EFI to a drive called 'SYSTEM'
6. Replaced files in the hard drive EFI with the files found in this repo under 'EFI-OpenCore'
7. Rebooted - it still tries to boot to the Windows recovery - in order to go into Mac I still need to boot from the USB. 

Am I missing a step here? Is there a link to the instructions for installing OpenCore to boot without USB?

Thanks!

**Comment by erdemoney** [comment]
You might need to change boot order in bios assuming that you did everything else correctly. However it doesn't seem that you're following the instructions here very closely because I don't see "OpenEFI" mentioned anywhere in this guide. I don't know if it's a Clover thing or what because I've never used Clover, but if you're trying to use Opencore and follow this guide then reference this section here to boot from your internal drive.

https://dortania.github.io/OpenCore-Post-Install/universal/oc2hdd.html#grabbing-opencore-off-the-usb

**Comment by DanielYawJr** [comment]
> I don't know if "OpenEFI" is a Clover thing or something but if you're trying to use Opencore and follow this guide then reference this section here and use `MountEFI` to boot from your internal drive.
> 
> https://dortania.github.io/OpenCore-Post-Install/universal/oc2hdd.html#grabbing-opencore-off-the-usb
> 
> tldr;
> 
> ```
> git clone https://github.com/corpnewt/MountEFI.git
> cd MountEFI
> ./MountEFI.command
> ```
> 
> Select USB
> Move EFI folder from USB's mounted EFI partition to desktop
> Eject EFI partition
> `./MountEFI.command`
> Select Macintosh HD
> Copy EFI folder from desktop to Macintosh HD's mounted EFI partition
> Reboot and test if it boots without USB
> 
> Also... if you're trying to use macOS as your only OS you should've wiped your entire drive and reformatted as APFS before you installed macOS. That should get rid of the Windows recovery on your drive as far as I know, so I'm not sure why you're even able to boot into that.

Just did that, and now it boots right to Mac! And sorry, meant to say MountEFI not OpenEFI,

This may be a bit of a dumb question... but how do I tell if I'm booting with Clover or OpenCore? My EFI folder has both a 'CLOVER' and 'OC' folder in it

**Comment by erdemoney** [comment]
AFAIK If you're using the EFI folder from this repo that includes opencore.efi you should be booting via opencore and it is safe to remove the "CLOVER" folder.

**Comment by DanielYawJr** [comment]
> AFAIK If you're using the EFI folder from this repo that includes opencore.efi you should be booting via opencore and it is safe to remove the "CLOVER" folder.

Perfect, yep, works well now. 

One more thing - in step 3, part 3, I copied the data from 'EFI-OpenCore' to replace the data on the hard drive's EFI folder - and when I do that, the system freezes during boot (at the apple logo screen) at around 50% - am I doing something wrong here, or is there error logs? 

Thanks again for your help!

**Comment by erdemoney** [comment]
This is most likely because you didn’t add the unmodded_bios_config.plist to config.plist. 

Also as a note, all of my answers have been based on thoroughly reading the guide (I’m new to this too), so I would recommend you do the same.

**Comment by DanielYawJr** [comment]
> This is most likely because you didn’t add the unmodded_bios_config.plist to config.plist.
> 
> Also as a note, all of my answers have been based on thoroughly reading the guide (I’m new to this too), so I would recommend you do the same.

Yep, that fixed it! And yep, you're right - I've been busy and glanced over some of this stuff, I need to take the time to read through it more throughly.
